src/raftstore/store/fsm/store.rs

```diff
@@ -467,6 +468,17 @@ impl<T: Transport, C: PdClient> Store<T, C> {
             warn!("set thread priority for raftstore failed, error: {:?}", e);
         }
 
+        {
+            // entry_gc thread
+            let (tx, rx) = mpsc::channel();
+            self.entry_gc_sender = Some(tx);
+            thread::spawn(move || loop {
+                if rx.recv().is_err() {
```

Comment:
maybe you should add some comments for that. If one doesn't know this issue, it would be confusing.

Reply:
Done